frontend/src/routes/(unauthenticated)/register/+page.svelte

```diff
@@ -4,8 +4,15 @@
   import t, { getLanguageCodeFromNavigator, locale } from '$lib/i18n';
   import { TitlePage } from '$lib/layout';
   import { register } from '$lib/user';
+  import { getSearchParamValues } from '$lib/util/query-params';
+  import { onMount } from 'svelte';
   import { z } from 'zod';
 
+  type RegisterPageQueryParams = {
+    name: string;
+    email: string;
+    locale: string;
+  };
```

Comment:
I agree on `locale`, but I do want to be able to support `name`. We'd have to modify the UI in the "Add member" modal to allow specifying the name of the user we want to invite, as well as their email address. We might even look for email address input in the form `Real Name <email@domain>` and parse it, as a bells-and-whistles kind of feature, so that if someone pastes in some address from their address book, we can extract the name from it.

Also, if the "Add Member" UI allowed managers to specify the name of the user they're adding, I could use that in the email template. Right now the email template is the only one that *doesn't* greet the recipient by name, and that feels weird.

I'll remove `locale` for now, and see whether we should remove `name` as well or whether we want to modify the "Add Member" UI to allow specifying the name of the member we're adding.

Reply:
Removed `locale` in commit 83144638. Let's discuss whether adding "Name" to the "Add Member" UI is worth doing, and how, before I resolve this conversation.

Reply:
Removed `name` in commit eec5af65.